static/app/constants/chartPalette.tsx

```diff
@@ -177,3 +177,82 @@ export const CHART_PALETTE = [
     '#f2b712',
   ],
 ] as const;
+
+export type ChartColorPalette = typeof CHART_PALETTE;
+type ValidLengthArgument =
+  | 0
+  | 1
+  | 2
+  | 3
+  | 4
+  | 5
+  | 6
+  | 7
+  | 8
+  | 9
+  | 10
+  | 11
+  | 12
+  | 13
+  | 14
+  | 15
+  | 16
+  | 17;
```

Comment:
we can compute this union with a `GetRange` helper. Have a look at this [TypeScript playground](https://www.typescriptlang.org/play/?#code/MYewdgzgLgBKA2IBOEYF4aYNoChOYHIBiAFjIDYB2EggGj32IFYAmEgQ0oFM6HCjybACYAOGvXwxi1EgDMRI3pOIjyTAIwiAnEsZEtIpgGZtu-u3ZMSCs1KIAjYEyaHbxYEKbkR6t0SGyXpTAflyqAAyU4aFC5MDk4nx2suqUWkw6Enqy1lY8WfyBWmwsfkVcRkxlHOwslGUs9pTqpVkAujDsqKCQUDg4UACeAA5cMADiXFAASuxgAOZcADwAcjBcAB5QXGBCqGAArgC29lxItDAAgutbO3swB2AA1mAgAO5gWB0YXwB86FcsAR4Dt5lAABYEDqbba7VBrAD8gMOJzOHQAXBMprMFssVhcsAA6YmXC6XIEghYQqFtX79IajGAANXY8AAlkIADKgiGXJDzY47WAYSYzOaLJYMrggWRwECIFAUnmQ2lAA)

Reply:
Good call, let me use that